### ISSUE TITLE
Show list view on mobile

### ISSUE THREAD

**Issue by niallthompson** [nicinabox/superslides#204]
Is it possible to change the view to a standard list view on mobile so that the images are stacked on top of each other so the user can scroll down?

At the moment because of the portrait ratio of a mobile device the images are being cropped too aggressively.

It would be great to trigger the gallery script at a certain breakpoint.

Thoughts?

**Comment by nicinabox** [comment]
I link this idea. Send a PR!

**Comment by niallthompson** [comment]
You could use something like https://github.com/WickyNilliams/enquire.js to run the script at a certain breakpoint but I'm not entirely sure how to do it.

**Comment by nicinabox** [comment]
Yep, you destroy the superslides object and take over with your own CSS at a breakpoint.

**Comment by niallthompson** [comment]
Like this:

enquire.register("screen and (min-width: 768px)", {
    match : $(function() {
    var $slides = $('#slides');

```
Hammer($slides[0]).on("swipeleft", function(e) {
  $slides.data('superslides').animate('next');
});

Hammer($slides[0]).on("swiperight", function(e) {
  $slides.data('superslides').animate('prev');
});

$slides.superslides({
  hashchange: true
});
```

  });
  });

```
},  
unmatch : function() {
  $('#slides').superslides('destroy');
}
```

});

Doesn't seem to work, the superslides never destroys.

**Comment by nicinabox** [comment]
Are you sure? It may not wipe out the inlined css, unfortunately, but it should [destroy the attached data](https://github.com/nicinabox/superslides/blob/0.6-stable/src/core_prototype.js#L91). It should be simple enough to loop over the slides and set `.attr('style', '')`.
